privacyidea/lib/token.py

```diff
@@ -221,7 +221,7 @@ def _create_token_query(tokentype=None, realm=None, assigned=None, user=None,
             sql_query = sql_query.filter(Token.resolver == user.resolver)
         (uid, _rtype, _resolver) = user.get_user_identifiers()
         if uid:
-            sql_query = sql_query.filter(Token.user_id == uid)
+            sql_query = sql_query.filter(Token.user_id == str(uid))
```

Comment:
Could there be any scenario, where we could run into an encoding error, when we use ``str`` - with cool uid's like "kölbel" or u"kölbel"?

Reply:
I think there might indeed be a problem with ``uidtype=dn`` and a unicode username, because the result of ``getUserId`` will be unicode then :-/ The ``getUserId`` documentation is not really correct.
... one more reason for Python 3 :-)